### PR TITLE
Google ldap doc updates

### PR DESCRIPTION
**Pull request by garci66**
# Description
Documentation updates for Google Workspace LDAP source. Makes it a bit clearer what needs to be provisioned / provides some example values

# Impacts
Documentation

# Delete branch after merge
YES 

# Checklist
(REQUIRED) - [yes, no or n/a]
- [yes] Document the feature

# NEWS file entries
(REQUIRED, but may be optional...)

### CODE REVIEW

File: docs/installation/google_workspace_ldap.asciidoc [hunk 1]
```diff
@@ -22,3 +22,15 @@
     ... Password: The access credentials password
     ... Client Certificate: The .crt file text from the downloaded certificate bundle
     ... Client Key: The .key file text from the downloaded certificate bundle
+  .. You will also need to configure several values for the Authentication Source:
```
Comment (julsemaan) [inline docs/installation/google_workspace_ldap.asciidoc]:
```suggestion
  .. You will also need to these properties for the Authentication Source:
```

File: docs/installation/google_workspace_ldap.asciidoc [hunk 2]
```diff
@@ -22,3 +22,15 @@
...
+        You might have to add `ou=Users,` as a prefix in some cases, so it would be `ou=Users,dc=example,dc=com` 
+    ... Scope: `Subtree`
+    ... Username Attribute: `uid` (unless you've heavily customized your Google Workspace directory)
+    ... Email Attribute: `mail` 
+    ... Associated Realms: You'll need to match a previously created realm which matches your `example.com` domain. This will let the system strip the domain part when searching for the user and also let the system know which source to use for which specific realms (the @example.com part of the username) are used for each source.
+  .. For Authentication and Administration rules, you can match against google group membership (if you have configured google to allow group membership access - this is done when creating the ldap client on the Google workspace configuration page on Google's side, not on PacketFence). In that case, you want to use the condtionn `memberOf`, a match of `equals` and the value of `cn=mygroupname,ou=Groups,dc=example,dc=com` if your group is called "mygroupname`. Keep in mind that nested group membership does not work via ldap for google workspace.
```
Comment (julsemaan) [inline docs/installation/google_workspace_ldap.asciidoc]:
```suggestion
  .. For Authentication and Administration rules, you can match against google group membership (if you have configured google to allow group membership access - this is done when creating the LDAP client on the Google workspace configuration page on Google's side, not on PacketFence). In that case, you will want to use the condition `memberOf`, a match of `equals` and the value of `cn=mygroupname,ou=Groups,dc=example,dc=com` if your group is called "mygroupname`. Keep in mind that nested group membership does not work via ldap for google workspace.
```

**Review by julsemaan — CHANGES_REQUESTED** [review]
Some minor details to fix and then I'm happy to merge

Thanks for your contribution!

**Comment by garci66** [comment]
You're welcome. Seeing the number of questions in the mailing list about google, I thought this could help.

**Pull request merged**